functionsChallenges.js

```diff
@@ -0,0 +1,74 @@
+function maxOrMin(max, x, y) {
+    if (max && x > y) {
```

Comment:
Really close - we want to return the max if `true` is passed in and the min if `false` is passed in. Example:

```
maxOrMin(true, 9, 7);
//=> 9

maxOrMin(false, 9, 7);
//=> 7
```

Try implementing these steps:

```js
function maxOrMin(max, x, y) {
  // figure out which number is max and which is min (store these as variables `var maxNum` and `var minNum`

  if (max) {
    // return maxNum
  } else {
    // return minNum
  }
}
```